packages/vkui/src/components/Button/Button.test.tsx

```diff
@@ -7,9 +7,9 @@ const ButtonTest = (props: ButtonProps) => <Button data-testid="custom-btn" {...
 const button = () => screen.getByTestId('custom-btn');
 
 describe('Button', () => {
-  baselineComponent(Button);
+  baselineComponent((props) => <Button {...props}>Button</Button>);
 
-  it('Component: Button is handled as a native button', () => {
+  it('[unit][a11y] Component: Button is handled as a native button', () => {
```

Comment:
```suggestion
  it('Component: Button is handled as a native button', () => {
```
префиксы ведь не нужны теперь